### DISCUSSION TITLE
Farm Bound NFTs (SFL to the Moon!)

### DISCUSSION THREAD

**Discussion by PotatoChipsss** [sunflower-land/sunflower-land#1152] [Ideas]
I noticed that new player that join don't really have much choice on NFTs to craft cause well most of them are sold out, introducing Farm Bound NFTs that give similar but weaker effects and every farm can only craft 1 which you can't trade or sell it

Example
1) Scarecrow (Bound) - 25 SFL 100 Wheat (Yield 10% more crops)
2) Tunnel Mole (Bound) - 30 Stone (Yield 10% more stone)
3) Golden Cauliflower (Bound) - 50 SFL 200 Cauliflower 30 Gold (Yield 50% more Cauliflower)

Benefits
Now new player have something to work towards and since there is no time limit, they can have their own sweet time working towards the goal instead of worrying Chicken Coop might already finished by the time he get enough egg

Bound & Unbound NFTs
They do not stack, so people that already have the NFTs wouldn't benefits from having it, this bound NFTs is just for new player to have a chance to be half as powerful as old player

**Comment by ilNomade** [comment]
It doesn't make much sense to me, otherwise the concept of a limited object is lost.

**Comment by Galphath** [comment]
I would be in for cosmetic bound nft that are also time dependant; for example a participation statue/trophy for an event like goblin vs humans one.  One could pick which ones ahow on the farm as if they were achievements
